helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -154,6 +154,14 @@ starrocksFESpec:
     storageSize: 10Gi
     # Setting this parameter can persist log storage
     logStorageSize: 5Gi
+  # mount other volumes if necessary.
+  # Note: please use storageSpec field for persistent metadata and log.
+  storageVolumes: []
```

Comment:
the user will be confused by this `storageVolumes` and `storageSpec`, what's the different when to use which one.

Reply:
yes, do you have any suggestions? 
storageVolumes can be configured with PVC and EmptyDir, or should we name it emptyDirVolume and only allow users to set EmptyDir?